### DISCUSSION TITLE
Directory too big

### DISCUSSION THREAD

**Discussion by fewirchen** [sepinf-inc/IPED#677] [Q&A]
Nassif or someone else who wants to help.

I ran the IPED 3.18.7 over an account with twenty files *.msg with about 10/20KB each one and the result, directories jre, lib and tools stayed about 600, 200 e 200MB respectively.

Is it normal?

**Comment by wladimirleite** [comment]
Hi @fewirchen!

As you can notice inspecting their contents, the folders you mentioned (jre, lib and tools) contain, respectively:
- A "compatible" Java JRE, 
- Several libraries used by IPED,
- External tools (mostly Windows binaries) also used by IPED.

With IPED version 3.18.7, I am getting here:
- jre: 189 MB
- lib: 231 MB
- tools: 218 MB.
So each of them should be close to 200 MB.

As IPED processed cases are "portable" (in the sense that the processed case won't depend on original IPED's installation folder to be opened), these folders are always copied to the case, and will be the same, regardless the amount of data you processed.

**Reply by lfcnassif** [reply]
Same here.
